docs/versioned-plugins/codecs.asciidoc

```diff
@@ -0,0 +1,6 @@
+:type: codec
+:type_uc: Codec
+
+include::include/plugin-intro.asciidoc[]
+
+include::codecs/rubydebug-index.asciidoc[]
```

Comment:
this is really easy to template and expand with the list of codecs retrieved from the logstash-plugins github org